xml/System.Runtime.InteropServices/SequenceMarshal.xml

```diff
@@ -176,9 +176,15 @@
         <typeparam name="T">To be added.</typeparam>
         <param name="reader">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Tries to read the given type out of the buffer if possible.
+Warning: this is dangerous to use with arbitrary structs - see remarks for full details.</summary>
+        <returns><see langword="true" /> if successful, <see langword="false" /> otherwise. <paramref name="value" /> will be default if failed (due to lack of space).</returns>
```

Comment:
Before I make suggestions to the language here, I don't understand what value being the default here means. This is either true or false, no? Are you trying to say that if the operation fails, value will contain the default value of T?

Reply:
Since `value` has a type T, depending on what type it is, it will be returned with a `default` value for that data type. I think that portion of the `returns` description should go in the description for `value`. I'll move it.

```suggestion
        <returns><see langword="true" /> if successful, <see langword="false" /> otherwise.</returns>
```